azkaban-common/src/main/java/azkaban/constants/ServerProperties.java

```diff
@@ -47,4 +47,7 @@
 
   public static final String IS_METRICS_ENABLED =
       "azkaban.is.metrics.enabled";
+
+  public static final String EXECUTOR_HOST = "executor.host";
```

Comment:
Could you add a comment here that this section of the constants are legacy configs?

The new configs should follow the naming convention of azkaban.server.<name> for server configs.